setup.py

```diff
@@ -75,6 +75,8 @@ def run(self):
     "win-unicode-console>=0.5; sys_platform == 'win32'",
     "pywin32>=225; sys_platform == 'win32'",
     "networkx>=2.1,<2.4",
+    "speedcopy @ git+https://github.com/rxxg/speedcopy@5",
```

Comment:
As @rxxg noted, need to wait for official release here, as otherwise it will break our pypi package, as it doesn't support direct gh dependencies 🙁

Reply:
Version 2.0.1 of speedcopy has been released and is available on pypi